examples/closing-channels/closing-channels.sh

```diff
@@ -7,6 +7,11 @@ sent job 3
 received job 3
 sent all jobs
 received all jobs
+no jobs to receive 0
 
-# The idea of closed channels leads naturally to our next
-# example: `range` over channels.
+# Be sure that channel is not closed when you read
```

Comment:
Let's leave this out, keeping the original text.
gobyexample tries not to be too tutorial-y describing all the possible edge cases